rust/schema.py

```diff
@@ -480,19 +480,19 @@ class InlineAsm(Expr):
 #         else_branch: Option<ExprId>,
 #     },
 
-class IfLet(Stmt):
-    pat: Pat
-    type_ref: optional[TypeRef]
-    initializer: optional[Expr]
-    else_: optional[Expr]
+class LetStmt(Stmt):
```

Comment:
Is this a [`let`](https://doc.rust-lang.org/std/keyword.let.html) or an [`if let`](https://doc.rust-lang.org/reference/expressions/if-expr.html#if-let-expressions)? The old name and the fact that it has an else branch makes it look like the latter. If that's true I think `IfLet` should still be in the name?

Reply:
That's what I thought too, but I was wrong. That's why I changed the name. The use of `else` is very rare, but I found some examples using QL:

https://github.com/rust-lang/rust/blob/304b7f801bab31233680879ca4fb6eb294706a59/library/alloc/src/vec/in_place_collect.rs#L322

Reply:
I see. Thanks for the explanation, I had no idea that a `let` could have and `else` branch. Looking it up, the docs state that "the `else` block must always diverge" so that's useful to be aware of for the CFG.

Reply:
> I had no idea that a `let` could have and `else` branch

oh, me neither!